Tests/scripts/configure_tests.py

```diff
@@ -888,9 +896,10 @@ def is_test_integrations_available(server_version, test_conf, conf, id_set):
             test_integration_ids = [test_integration_ids]
         if not is_test_uses_active_integration(test_integration_ids, conf):
             return False
-        # check integrations from/toversion is valid with server_version
+        # check if any integration from/toversion is valid with server_version
         integrations_set = id_set.get('integrations', [])
-        if any(extract_matching_object_from_id_set(integration_id, integrations_set, server_version) is None for integration_id in
+        if all(extract_matching_object_from_id_set(integration_id, integrations_set, server_version) is None for
```

Comment:
should be any.
if one of the integration is missing - integration is not available